postgres/cds-plugin.js

```diff
@@ -1,9 +1,10 @@
-const cds = require('@sap/cds/lib')
+const cds = require('@sap/cds')
 
 if (!cds.env.fiori.lean_draft) {
   throw new Error('"@cap-js/postgres" only works if cds.fiori.lean_draft is enabled. Please adapt your configuration.')
 }
 
-if (cds.cli.command === 'build') {
-  module.exports = require('./lib/build')
-}
+cds.build?.register?.('postgres', {
+  impl: __dirname + '/lib/build.js',
+  taskDefaults: { src: cds.env.folders.db, dest: 'pg' }

```

Comment:
I think configuring a default for the `dest` folder is not a good idea as this will create the build output in the wrong folder for Java apps. Instead the `pg` folder has to be passed to the corresponding `copy.to` or `write.to` methods 
Node.js: `./gen/pg/**` (build.target folder is `gen`)
Java: `./db/pg/gen/db/**` (build.target folder is `.`)

**Note:** Paths are resolved in the following way - `<cds.root>/<build.target>/<task.dest|task.src>/gen`